Automattic/jetpack
# Update/change go back behavior in my jetpack

## Proposed changes:

* This PR changes the behavior of the "Go back" button on My Jetpack product interstitial pages to always return to the My Jetpack homepage
* Previously, it would just go back in history, which produced some odd edge cases

![Screenshot 2024-04-01 at 7 37 09 PM](https://github.com/Automattic/jetpack/assets/18016357/5e71b1cc-7d32-420c-bac8-2f939fe669fc)

## Testing instructions:






* Load up My Jetpack on a test site with this PR
* Click "Learn more" on the backup, card - this takes you to: `/wp-admin/admin.php?page=my-jetpack#/add-backup`
* Now, manually change the URL to `/wp-admin/admin.php?page=my-jetpack#/add-protect` in the address bar and press enter.
* From the protect interstitial, click "go back" in the upper left
* This should send you back to the main My Jetpack page
